src/utils.js

```diff
@@ -21,7 +21,9 @@ export function getNodeName(node) {
 }
 
 export function getEventName(attrName) {
-  return attrName.replace('on', '').toLowerCase();
+  let eventName = attrName.replace('on', '').toLowerCase();
+  if(eventName.endsWith('capture')) return eventName.replace('capture', '');
```

Comment:
I could add the capture replace in the same code itself, but it would be better if we only remove it my removing capture from the end via `substring`

Just not sure, if thats reasonable for some event to have capture in its name

Reply:
Actually there are two, gotpointercapture and lostpointercapture.
https://developer.mozilla.org/en-US/docs/Web/Events

I don't see preact handling it. https://github.com/preactjs/preact/blob/master/src/diff/props.js#L92

Have to check react if they handle it differently

Reply:
Also you can replace on and capture together using regex.
```js
.replace(/^on(.*?)(Capture$|$)/, '$1')
```